suzuki/reactions_suzuki.py

```diff
@@ -0,0 +1,18 @@
+from chython import smiles
+from chython.reactor import reactions
+from itertools import product
+with open('reag1.txt') as f:
+    for line in f:
+        a = line.strip()
+        a = smiles(a)
+with open('reag2.txt') as f:
+    for line in f:
+        b = line.strip()
+        b = smiles(b)
+try: 
+    prod = str(next(reactions.suzuki(a, b)))
+    product = prod.split(">>")
+    with open('product.txt','w') as f:
+        f.write(product[1])
```

Comment:
Вот этот блок лучше вынести на после except. В try, except оборачивай только тот участок кода, в котором может возникнуть ошибка, обычно это одна конкретная строчка кода

```suggestion
    with open('product.txt','w') as f:
        f.write(product[1])
```